### PR TITLE
Docker compose

### PR DESCRIPTION
**Pull request by Dalidovich**

### CODE REVIEW

File: HCL.IdentityServer.API.BLL/Services/RegistrationService.cs [hunk 1]
```diff
@@ -26,22 +26,23 @@ public RegistrationService(ILogger<RegistrationService> logger, IOptions<JWTSett
             _options = options.Value;
             _accountService = accountService;
         }
-        public async Task<BaseResponse<(string, Guid)>> Registration(AccountDTO DTO)
+        public async Task<BaseResponse<AuthDTO>> Registration(AccountDTO DTO)
```
Comment (KatsiarynaAnashkevich) [inline HCL.IdentityServer.API.BLL/Services/RegistrationService.cs:29]: Do you have global excrption handler?

File: HCL.IdentityServer.API.BLL/Services/RegistrationService.cs [hunk 2]
```diff
@@ -26,22 +26,23 @@ public RegistrationService(ILogger<RegistrationService> logger, IOptions<JWTSett
...
-                        Message = "Account with that login alredy exist"
+                        Message = "Account with that login alredy exist",
+                        StatusCode = StatusCode.AccountExist
                     };
                 }
                 CreatePasswordHash(DTO.Password, out byte[] passwordHash, out byte[] passwordSalt);
                 var newAccount = new Account(DTO, Convert.ToBase64String(passwordSalt), Convert.ToBase64String(passwordHash));
                 newAccount = (await _accountService.CreateAccount(newAccount)).Data;
-                return new StandartResponse<(string, Guid)>()
+                return new StandartResponse<AuthDTO>()
```
Comment (KatsiarynaAnashkevich) [inline HCL.IdentityServer.API.BLL/Services/RegistrationService.cs:45]: empty line before return

File: HCL.IdentityServer.API.BLL/Services/RegistrationService.cs [hunk 3]
```diff
@@ -51,38 +52,38 @@ public async Task<BaseResponse<(string, Guid)>> Registration(AccountDTO DTO)
...
                 {
-                    return new StandartResponse<(string, Guid)>()
+                    return new StandartResponse<AuthDTO>()
                     {
                         Message = "account not found"
                     };
                 }
                 string token = GetToken(account.Data);
-                return new StandartResponse<(string, Guid)>()
+                return new StandartResponse<AuthDTO>()
```
Comment (KatsiarynaAnashkevich) [inline HCL.IdentityServer.API.BLL/Services/RegistrationService.cs:77]: empty line before return

File: HCL.IdentityServer.API.BLL/Services/RegistrationService.cs [hunk 4]
```diff
@@ -51,38 +52,38 @@ public async Task<BaseResponse<(string, Guid)>> Registration(AccountDTO DTO)
...
-                    Data = (token, (Guid)account.Data.Id),
+                    Data = new AuthDTO(token, (Guid)account.Data.Id),
                     StatusCode = StatusCode.AccountAuthenticate
                 };
             }
             catch (Exception ex)
             {
                 _logger.LogError(ex, $"[Authenticate] : {ex.Message}");
-                return new StandartResponse<(string, Guid)>()
+                return new StandartResponse<AuthDTO>()
```
Comment (KatsiarynaAnashkevich) [inline HCL.IdentityServer.API.BLL/Services/RegistrationService.cs:86]: empty line before return

File: HCL.IdentityServer.API.Domain/DTO/AccountDTO.cs [hunk 5]
```diff
@@ -8,7 +9,11 @@ namespace HCL.IdentityServer.API.Domain.DTO
 {
     public class AccountDTO
     {
+        [Required(AllowEmptyStrings = true,ErrorMessage = "Need Login")]
         public string Login { get; set; }
+
```
Comment (KatsiarynaAnashkevich) [inline HCL.IdentityServer.API.Domain/DTO/AccountDTO.cs:14]: Only one empty line should be

File: HCL.IdentityServer.API.Domain/DTO/AuthDTO.cs [hunk 6]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HCL.IdentityServer.API.Domain.DTO
+{
+    public record AuthDTO(string JWTToken,Guid accountId);
```
Comment (KatsiarynaAnashkevich) [inline HCL.IdentityServer.API.Domain/DTO/AuthDTO.cs:9]: Please, use class or record every. One type for better code styling

File: HCL.IdentityServer.API/Controllers/IdentityServerController.cs [hunk 7]
```diff
@@ -20,54 +22,55 @@ public IdentityServerController(ILogger<IdentityServerController> logger, IRegis
...
             var resourse = await _registrationService.Authenticate(accountDTO);
-            if (resourse.Data.Item1 == null)
+            if (resourse.StatusCode == Domain.Enums.StatusCode.AccountAuthenticate)
             {
-                return Results.NoContent();
-            }
-            else
-            {
-                return Results.Json(new { token = resourse.Data.Item1, id = resourse.Data.Item2 });
+                return Ok(Results.Json(resourse.Data));
```
Comment (KatsiarynaAnashkevich) [inline HCL.IdentityServer.API/Controllers/IdentityServerController.cs:35]: You can pass resourse.Data without Results.Json

File: HCL.IdentityServer.API.BLL/Services/RegistrationService.cs (hunk 1 above)
Comment (Dalidovich) [reply HCL.IdentityServer.API.BLL/Services/RegistrationService.cs:29]: I don't use global exception handler in identity server

**Pull request merged**